include/deal.II/grid/grid_in.h

```diff
@@ -456,6 +455,46 @@ class GridIn
   void read_tecplot (std::istream &in);
 
   /**
+   * Read in a file supported by Assimp, and generate a Triangulation
+   * out of it.  If you specify a @p mesh_index, only the mesh with
+   * the given index will be extracted, otherwise all meshes which are
+   * present in the file will be used to generate the Triangulation.
+   *
+   * This function can only be used to read two-dimensional meshes (possibly
+   * embedded in three dimensions). This is the standard for graphical software
+   * such as blender, or 3D studio max, and that is what the original Assimp
+   * library was built for. We "bend" it to deal.II to support complex
+   * co-dimension one meshes and complex two-dimensional meshes.
+   *
+   * If @p remove_duplicates is set to true (the default), then
+   * duplicated vertices will be removed if their distance is lower
+   * than @p tol.
+   *
+   * Only the elements compatible with the given dimension and spacedimension
+   * will be extracted from the mesh, and only those elements that are
+   * compatible with deal.II are supported. If you set
+   * `ignore_unsupported_element_types`, all the other element types are simply
+   * ignored by this algorithm. If your mesh contains a mixture of triangles
+   * and quadrilaterals, for example, only the quadrilaterals will be
+   * extracted. Your mesh may not make any sense if you are mixing compatible
```

Comment:
To avoid confusing between the use of "your mesh" in this sentence and the one in the previous sentence, how about writing "The resulting mesh (as represented in the Triangulation object) may not make ..." in the second sentence?